pkgs/development/python-modules/torchmetrics/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, packaging
+, py-deprecate
+, pytorch
+}:
+
+let
+  pname = "torchmetrics";
+  version = "0.8.1";
+in
+buildPythonPackage {
+  inherit pname version;
+
+  propagatedBuildInputs = [
+    packaging
+    py-deprecate
+  ];
+
+  # Let the user bring their own instance
+  buildInputs = [
+    pytorch
+  ];
+
+  src = fetchFromGitHub {
+    owner = "PyTorchLightning";
+    repo = "metrics";
+    rev = "v${version}";
+    hash = "sha256-AryEhYAeC97dO2pgHoz0Y9F//DVdX6RfCa80gI56iz4=";
+  };
+
+  dontUseSetuptoolsCheck = true;
```

Comment:
We at least need a pythonImportsCheck and better some other checkPhase.

Reply:
Import checks still do run

Reply:
Enabled the test suite for torchmetrics (most of it)

Reply:
Takes ages to execute though

Reply:
Can we enable some parallelism or subset of tests?

Reply:
I can add more `disabledTestPaths` but that'd get ugly. I could move the test suite to `passthru.tests` and leave just imports check.

...would that `pytestCheckHook` had `enabledTests` alternative to `disabledTests`